src/docs/asciidoc/en/firebirddocs/nbackup/firebird-nbackup.adoc

```diff
@@ -196,10 +196,22 @@ To make a full database backup, the command syntax is:
 nbackup [-U _user_ -P _password_] -B 0 _database_ [_backupfile_]
 ----
 
-For instance:
+For instance, assuming the database is located in C:\Data, and nbackup.exe is in the search of path Windows:
+
+----
+C:\Data>nbackup -B 0 inventory.fdb inventory_1-Mar-2006.nbk
+----
+
+Or, if Firebird (from version 2.5) is running on non-standard port, in this example, 3051:
+----
+C:\Data>nbackup -B 0 localhost/3051:C:\Data\inventory.fdb C:\Data\inventory-level-0-Jul-2020.nbk -user SYSDBA -pass masterkey
+----
 
+In Firebird 3.0+, in a case of successful completing the backup, the nbackup will print the short statistics:
```

Comment:
I'd suggest using _Firebird 3 and higher_ instead of _Firebird 3.0+_, alternatively I'd recommend using `{plus}` instead of just `+`, because `+` can have special meaning in AsciiDoctor. In this case it works, but if another `+` ends up on the same line it will become a pass-through span.